ios/StatusIm/AppDelegate.m

```diff
@@ -50,6 +50,9 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   RCTRootView *rootView = [[RCTRootView alloc] initWithBridge:bridge
                                                       moduleName:@"StatusIm"
                                                initialProperties:nil];
+  if (@available(iOS 13.0, *)) {
+      rootView.overrideUserInterfaceStyle = UIUserInterfaceStyleLight;
+  }
```

Comment:
Maybe a comment explaining this hack would be a good idea?

Reply:
It looks kind of self explanatory to me, what exactly would you like to see there? I will add it.

Reply:
Comments aren't supposed to explain what code does, they are supposed to explain why we want what it does. Why do we want this hack? It's not clear from the code.

Reply:
We will have a link to the original GH issue in the commit, should be sufficient enough imo, but well I can put that link to the code as well if necessary.

Reply:
Yes, usually `git blame` is all you need, but if something is a "temporary solution" as you state in the PR description than I think it's only fair to indicate that with a comment. It's too easy to forget about stuff like that and leave it there without a clear indication that it's a temporary hack. But that's just my preference.